includes/Admin/Sync_Page.php

```diff
@@ -348,13 +386,17 @@ private static function output_sync_with_square_modal_template() {
 				<div class="wc-backbone-modal-content">
 					<section class="wc-backbone-modal-main" role="main">
 						<header class="wc-backbone-modal-header">
-							<h1><?php esc_html_e( 'Sync products with Square', 'woocommerce-square' ); ?></h1>
+							<h1><?php esc_html_e( 'Sync existing products with Square', 'woocommerce-square' ); ?></h1>
```

Comment:
```suggestion
							<h1><?php esc_html_e( 'Sync products with Square', 'woocommerce-square' ); ?></h1>
```

Please see comment below.